### PR TITLE
feat(markup): add helix markup properties to the html root node

### PR DESCRIPTION
**Pull request by ramboz**
Expose the html and body tag attributes in the HTL template.

Requires: https://github.com/adobe/helix-pipeline/pull/796

### CODE REVIEW

File: src/html.pre.js [hunk 1]
```diff
@@ -54,6 +68,7 @@ function pre(context) {
       });
     }
     $(document.body).children().wrapAll(div);
+    $sections = $(document.body).children('div');
```
Comment (ramboz) [inline src/html.pre.js:71]: When there are no sections, we add a fake `div` wrapper, but didn't update the `sections` object.

**Comment by rofe** [comment]
I've seen it working, but honestly I don't really get the "single child on the body element" part - isn't there always at least a header, main and footer element in the body in Helix Pages?

**Comment by ramboz** [comment]
@rofe The single child use case is to handle the case where we `wrap` the content via `helix-markup`
This introduces an artificial wrapping div that we are removing here and merging on the body element directly, and then merging again on the html element in the HTL.

Kind of a double-hack to achieve the desired outcome. Not sure I'm really convinced by the feature to be honest.

**Pull request merged**